addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16376,7 +16386,7 @@ msgstr ""
 
 #: system/settings/rbp.xml
 msgctxt "#37024"
-msgid "Select this if the audio out connection only supports multichannel audio as Dolby Digital 5.1, this allows multichannel audio such as AAC5.1 or FLAC5.1 to be listened to in 5.1 surround sound. Note - Not recommended on Pi as this requires a lot of CPU."
+msgid "Select this if the audio out connection only supports multichannel audio as Dolby Digital 5.1, this allows multichannel audio such as AAC 5.1 or FLAC 5.1 to be listened to in 5.1 surround sound. Note: Not recommended on Pi as this requires a lot of CPU."
```

Comment:
Good catch, I thought since one or more places added AC3/E-AC3 all places should have it, good catch.